great_expectations/core/usage_statistics/usage_statistics.py

```diff
@@ -357,6 +357,23 @@ def edit_expectation_suite_usage_statistics(data_context, expectation_suite_name
     return payload
 
 
+def add_datasource_usage_statistics(data_context, name, **kwargs):
+    try:
+        data_context_id = data_context.data_context_id
+    except AttributeError:
+        data_context_id = None
+
+    datasource_anonymizer = DatasourceAnonymizer(data_context_id)
```

Comment:
Can this use the handler's anonymizer to avoid instantiating a new one?

Reply:
Addressed. Thanks to @roblim!